packages/read-model/.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: php
+
+php:
+    - 7.1
+    - 7.2
+    - 7.3
+
+cache:
+    directories:
+        - ~/.composer/cache
+
+before_install:
+    - echo "extension = redis.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+
+install:
+    - composer install
+
+script:
+    - php vendor/bin/parallel-lint ./src ./tests
+    - php vendor/bin/ecs check --verbose ./src ./tests
```

Comment:
hello we have created [this issue #880](https://github.com/shopsys/shopsys/issues/880) for removing --verbose from travis. I am not sure whether it make sense to create new scripts with that parameter set.